### DISCUSSION TITLE
Installation on Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by sitadrost** [abraunegg/onedrive#1360] [Q&A] [answered]
Recently I started using OneDrive client for Linux, which I installed on Ubuntu 20.04 following the instructions [here](https://gist.github.com/starlinq/0f98c6d9339497bb8ac42d67f66f60eb) (for further details see below).
According to @abraunegg the instructions I used are wrong; I should have used [these](https://github.com/abraunegg/onedrive/blob/master/docs/INSTALL.md) instructions instead.

So now I wondered: should I expect any kind of trouble if I keep using my current installation? Would it be a good idea for me to uninstall the whole thing and start over, using the correct instructions this time?

```
Configuration file successfully loaded
onedrive version                       = v2.4.10-16-g6bec0dd
Config path                            = /home/sita/.config/onedrive
Config file found in config path       = true
Config option 'check_nosync'           = false
Config option 'sync_dir'               = /home/sita/OneDrive
Config option 'skip_dir'               = 
Config option 'skip_file'              = ~*|.~*|*.tmp
Config option 'skip_dotfiles'          = false
Config option 'skip_symlinks'          = false
Config option 'monitor_interval'       = 300
Config option 'min_notify_changes'     = 5
Config option 'log_dir'                = /var/log/onedrive/
Config option 'classify_as_big_delete' = 1000
Config option 'upload_only'            = false
Config option 'no_remote_delete'       = false
Config option 'remove_source_files'    = false
Config option 'sync_root_files'        = false
Selective sync 'sync_list' configured  = false
Business Shared Folders configured     = false


```

**Comment by abraunegg** [comment, marked as answer]
@sitadrost 
> v2.4.10-16-g6bec0dd

You are running a 'newish' version of the client - however, because of the Ubuntu packaging mess - you will need to 'manually' install the latest client every month (new releases are ~monthly) - otherwise, you will have an old client, and if you encounter issues, I do not support older client versions - only clients >= current release.

**Reply by sitadrost** [reply]
Thanks, that makes sense. I'll start over from scratch, using the correct instructions this time.
